### ISSUE TITLE
"Load Setup" Button causes crash

### ISSUE THREAD

**Issue by joefowler** [usnistgov/quantum-sensors-tdm#9]
**[Original report](https://bitbucket.org/nist_microcal/nist-qsp-tdm/issues/9) by Dark Tyr (Bitbucket: [darktyr](https://bitbucket.org/darktyr), GitHub: [darktyr](https://github.com/darktyr)).**

----------------------------------------

With the GUI fully loaded, clicking on the “Load Setup” button and selecting a valid file \(even the one that was already loaded from the command line\) it crashes with the below error:

```python
Traceback (most recent call last):
  File "/home/pcuser/anaconda3/envs/cryoQA/lib/python3.10/site-packages/nistqsptdm-0.1.0-py3.10.egg/cringe/cringe.py", line 2346, in loadSettings
    f = open(self.load_filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: "('/home/pcuser/Documents/NGE_Set_Up_Files/CryoQA/SUMO1/SSA/OL_emulation/EmuBrittany_phase_2_25may22.pkl', 'All Files (*)')"
```

‌

Minor bug, but we do use the button on the CryoQA system when changing the testing phase.

**Comment by NanoExplorer** [comment]
Fixed by 514b45a ?

**Comment by ggggggggg** [comment]
Seems pretty likely.

**Comment by joefowler** [comment]
Yes. Fixed by 514b45a.
